.changelog/unreleased/breaking-changes/736-proto-enum-rename.md

```diff
@@ -0,0 +1,6 @@
+- `[abci]` Change the proto-derived enum type and constant aliases to the
```

Comment:
it would be nice to have an example here

Reply:
Added in #1707